libs/ui-components/src/components/Device/DevicesPage/DeviceToolbarFilters.tsx

```
@@ -41,15 +41,28 @@
 }) => {
   const { t } = useTranslation();
+  const regexp = React.useMemo(() => new RegExp(`(${filterText})`, 'g'), [filterText]);
+
+  const searchHighlighter = React.useCallback(
+    (text: string) => {
+      return text
+        .split(regexp)
+        .filter(Boolean)
+        .map((part, index) => (part.trim() === filterText ? <strong key={`part-${index}`}>{part}</strong> : part));
+    },
+    [filterText],
```

ESLint

[error] React Hook React.useCallback has a missing dependency: 'regexp'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)